components/navbar.js

```diff
@@ -0,0 +1,208 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Menu from "@mui/material/Menu";
```

Comment:
Not strictly necessary, but I think it looks more clean if you import all those components in the following way:
`import { AppBar, Box, Toolbar, IconButton, Typography, Menu } from '@mui/material';`
It works the same way (you can add as many components as you want from the same path).

But it is more of a styling thing, just for you to know it's possible, in case you didn't.

Reply:
Thanks Rony, I didn't know that!